.github/scripts/test_PR_against_release.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -eo pipefail
+export PATH=/opt/conda/bin:$PATH
+
+# write log as github Action artifact
+mkdir artifacts
+echo run in illumina mode with defaults >> artifacts/test_artifact.log
+
+# run current pull request code
+singularity --version
+NXF_VER=20.03.0-edge nextflow run ./main.nf \
+       -profile singularity \
+       --directory $PWD/.github/data/ \
+       --illumina \
+       --prefix test
+cp .nextflow.log artifacts/
+# save work dir and results for following tests
+cp -r results results_singularity_profile
+cp -r work work_singularity_profile
+
+# run tests against previous previous_release to compare outputs 
+git clone https://github.com/wtsi-team112/ncov2019-artic-nf.git previous_release 
+cd previous_release
+git checkout 42a79999f11304671b9d22e4959b0167b2130944 
+# the github runner only has 2 cpus available, so replace for that commit required:
+sed -i s'/cpus = 4/cpus = 2/'g conf/resources.config
+ln -s ../*.sif ./
```

Comment:
A previous version will likely have a different environment, so its sif would need re-creating. It's an important thing to test that new sif works as well as old sif. But maybe this can be ignored since the problem should go away when containers are in dockerhub.

Reply:
ok, yes at the moment it would also add 25 minutes to the tests.. Will ignore for now.